### PR TITLE
Add the OCM service

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #651 (New Service: OCM)

Adds the new OCM service to teh documentation.

### CODE REVIEW

File: modules/ROOT/pages/deployment/services/s-list/ocm.adoc [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+In order to do so, a request is sent to the sciencemesh API. The generated token is passed on to the receiver, who will then accept the invitation. As a result, remote users will be added on both sides.
+
+The data backend for the workflow, not the data itself, is configurable. The currently only supported backend is `json` which stores the data in a json file on disk.
+
+== Creating Shares
+
+OCM Shares are currently created using the xref:{s-path}/ocs.adoc[OCS] API, just like regular shares.
+
+The data backend for the workflow, not the data itself, is configurable. The currently only supported backend is `json` which stores the data in a json file on disk.
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/services/s-list/ocm.adoc:77]: repeats line 71

File: modules/ROOT/pages/deployment/services/s-list/ocm.adoc [hunk 2]
```diff
@@ -0,0 +1,81 @@
+= OCM Configuration
+:toc: right
+
+:service_name: ocm
+:no_second_tab: true
+//:no_third_tab: true
+
+:description: The Infinite Scale ocm service provides federated sharing functionality based on the sciencemesh and ocm HTTP API's.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/ocm.adoc:8]:
```suggestion
:description: The Infinite Scale OCM service provides federated sharing functionality based on the ScienceMesh and OCM HTTP APIs.
```

File: modules/ROOT/pages/deployment/services/s-list/ocm.adoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+* OCM listens on port 9280 by default.
+
+== Trust Between Instances
+
+IMPORTANT: For security reasons and data protection, invitations are limited to trusted instances only. These have to be defined by the administrator _before_ setting up any federation.
+
+The OCM service implements an invitation workflow for _trusted_ instances when creating federated shares.
+
+The list of trusted instances is managed via a configuration file. The currently only supported backend is `json`, which stores the list of trusted instances in a json file on disk.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/ocm.adoc:31]:
```suggestion
The list of trusted instances is managed via a configuration file. The only supported backend is currently `json`, which stores the list of trusted instances in a json file on disk.
```

File: modules/ROOT/pages/deployment/services/s-list/ocm.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                },
+                "api_version": "0.0.1",
+                "host": "example.com:port"
+            }
+        ]
+    },
+]
+----
+
+Note that there is no limit in adding multiple trusted providers.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/ocm.adoc:63]:
```suggestion
Note that there is no limit to adding multiple trusted providers.
```

File: modules/ROOT/pages/deployment/services/s-list/ocm.adoc [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+== Invitation Workflow
+
+Before sharing a resource with a remote user, this user has to be invited by the sharer.
+
+In order to do so, a request is sent to the sciencemesh API. The generated token is passed on to the receiver, who will then accept the invitation. As a result, remote users will be added on both sides.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/ocm.adoc:69]:
```suggestion
In order to do so, a request is sent to the ScienceMesh API. The generated token is passed on to the receiver, who will then accept the invitation. As a result, remote users will be added on both sides.
```

File: modules/ROOT/pages/deployment/services/s-list/ocm.adoc [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+The data backend for the workflow, not the data itself, is configurable. The currently only supported backend is `json` which stores the data in a json file on disk.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/ocm.adoc:71]:
```suggestion
The data backend for the workflow, not the data itself, is configurable. The only supported backend is currently `json` which stores the data in a json file on disk.
```

File: modules/ROOT/pages/deployment/services/s-list/ocm.adoc (hunk 1 above)
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/ocm.adoc:77]:
```suggestion
The data backend for the workflow, not the data itself, is configurable. The only currently supported backend is `json` which stores the data in a json file on disk.
```

**Review by EParzefall — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**